### PR TITLE
Footer v1

### PR DESCRIPTION
**Pull request by Mamoanwar97**
Check this and tell me your opinion, it just missing your mail link because it wasn't provided 😅  #19

### CODE REVIEW

File: src/components/atoms/ClickableIcon.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from "styled-components";
+import {Link} from "gatsby";
+
+const Icon = styled.img`
+    height: 25px;
```
Comment (fridaland) [inline src/components/atoms/ClickableIcon.js]: Is it possible to adjust this and other indentation to 2 spaces and not 4? Would be great to keep spacing consistent to the rest of the code! 

```suggestion
const Icon = styled.img`
  height: 25px;
```

File: src/components/atoms/ClickableIcon.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from "styled-components";
+import {Link} from "gatsby";
```
Comment (fridaland) [inline src/components/atoms/ClickableIcon.js:3]:
```suggestion
import { Link } from "gatsby";
```

File: src/components/atoms/ClickableIcon.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const Icon = styled.img`
+    height: 25px;
+`
+
+function IconsPanel({link, img, href, alt}) {
+    return href? (
+        <a target = "blank" href = {href} >
```
Comment (fridaland) [inline src/components/atoms/ClickableIcon.js]: Is this to handle the email icon without a link? 
Apologies for not including the link for this! You can use `info@inthefight.org`

File: src/components/templates/layout.js [hunk 4]
```diff
@@ -15,7 +15,11 @@ const Layout = () => {
   `)
 
   return (
-    <NavBar siteTitle={data.site.siteMetadata.title} />
+    <React.Fragment>
+      <NavBar siteTitle={data.site.siteMetadata.title} />
+        {children}
+      <Footer />
+    </React.Fragment>
```
Comment (fridaland) [inline src/components/templates/layout.js]:
```suggestion
    <>
      <NavBar siteTitle={data.site.siteMetadata.title} />
        {children}
      <Footer />
    </>
```

File: src/pages/contact.js [hunk 5]
```diff
@@ -4,8 +4,9 @@ import Layout from "../components/templates/layout"
 const ContactUs = () => {
   return (
     <>
-      <Layout />
-      Contact
+      <Layout>
+        Contact
+      </Layout>
```
Comment (fridaland) [inline src/pages/contact.js:9]: Nice! 🎉

File: src/pages/events.js [hunk 6]
```diff
@@ -13,17 +13,18 @@ class Events extends Component {
...
-          const { id, title, datetime, description } = event.node
-          return (
-            <div key={id} >
-              <Title text={title} />
-              <DateField date={datetime} />
-              <Description text={description.description} />
-            </div>
-          )
-        })}
+        <Layout>
```
Comment (fridaland) [inline src/pages/events.js:16]: Looks like an auto-formatting change. Thoughts on removing these types of changes? :)

**Review by fridaland — COMMENTED** [review]
This is looking great! Thanks for your contribution. Just have a couple thoughts/questions.

File: src/components/atoms/ClickableIcon.js (hunk 3 above)
Comment (Mamoanwar97) [reply src/components/atoms/ClickableIcon.js]: nah this was an option if you want to link to an outside site you use the href prop, and if you want to navigate in your site you use a link prop

okay I will add it now

Comment (fridaland) [reply src/components/atoms/ClickableIcon.js]: Got it! Thanks for the clarification

File: src/pages/contact.js (hunk 5 above)
Comment (Mamoanwar97) [reply src/pages/contact.js:9]: Thank you ^^

**Comment by Mamoanwar97** [comment]
check it now, i merged your comments with some responsivity fixes to look good in different screen sizes

This how should it looks on mobile :

screencapture-localhost-8000-events-2020-10-24-00_00_08

File: src/components/atoms/ClickableIcon.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from "styled-components";
+import {Link} from "gatsby";
+
+const Icon = styled.img`
+  height: 25px;
+`
+/*
+  USE link prop to navigate inside the site itself,
```
Comment (fridaland) [inline src/components/atoms/ClickableIcon.js:9]: Thanks for the clarifying comment!

**Review by fridaland — APPROVED** [review]
@mamoanwar97 Awesome this is ready to go! The only thing worth mentioning is that it's ideal to create a dedicated branch for your issue rather than committing straight into master. 
Great job, we really appreciate your contribution!

**Comment by Mamoanwar97** [comment]
Thank you for your feedback, really appreciated and good luck with your project 😄

**Pull request merged**